scripts/run_example.sh

```diff
@@ -7,42 +9,49 @@ export ENV="GREETING=hello"
 # Specify the message to send to the lambda
 export MESSAGE="{\"target\": \"world\"}"
 
-export DEFAULT_REGION=ap-southeast-2
+DCRUN="docker-compose run --rm"
+AWSLOCAL="$DCRUN awslocal"
+
+# Restart the localstack daemon, to clear state.
+docker-compose down
+docker-compose up -d localstack
 
-# Restart the localstack daemon
-localstack stop
-localstack start -d --no-banner
-localstack wait
+# TODO: cleanly wait for localstack to come up
+sleep 5
 
 # Setup the queue
 export QUEUE_NAME=test-queue
-awslocal sqs create-queue --queue-name="$QUEUE_NAME" --attributes '{ "VisibilityTimeout": "240" }' > /dev/null
-QUEUE_URL=$(awslocal sqs list-queues | jq -r '.QueueUrls[0]')
-QUEUE_ARN=$(awslocal sqs get-queue-attributes --queue-url="$QUEUE_URL" --attribute-names QueueArn | jq -r '.Attributes.QueueArn')
+$AWSLOCAL sqs create-queue --queue-name="$QUEUE_NAME" --attributes '{ "VisibilityTimeout": "240" }'
+QUEUE_URL=$($AWSLOCAL sqs list-queues | jq -r '.QueueUrls[0]')
+QUEUE_ARN=$($AWSLOCAL sqs get-queue-attributes --queue-url="$QUEUE_URL" --attribute-names QueueArn | jq -r '.Attributes.QueueArn')
 export QUEUE_URL QUEUE_ARN
 
-# Build the app and bundle it into a zip file
-if [ -z "$CI" ]; then
-   cargo b --example $EXAMPLE --target x86_64-unknown-linux-musl
-   cp target/x86_64-unknown-linux-musl/debug/examples/$EXAMPLE bootstrap && zip lambda.zip bootstrap && rm bootstrap
-else
-   cargo b --example $EXAMPLE
-   cp target/debug/examples/$EXAMPLE bootstrap && zip lambda.zip bootstrap && rm bootstrap
-fi
+# Build the app and bundle it into a zip file.
+# Need to strip debuginfo to keep it under maximum lambda file size.
+$DCRUN -e RUSTFLAGS="-C strip=debuginfo" cargo build --example $EXAMPLE --target x86_64-unknown-linux-musl
```

Comment:
This is a bit gross, but the default debug build was exceeding the size limit on a lambda, and I didn't want to spend the time always compiling in release mode for a smaller binary.